src/packages.jl

```diff
@@ -408,6 +408,66 @@ function build(name::String; quiet::Bool = false,
   return res
 end
 
+"""
+    test(name::String)
+
+Return a boolean indicating if the GAP package with name `name` succeeds
+in running its tests.
+
+It is inteded to be used with the `@test` macro from the `Test` package.
+
+The function uses [the function `TestPackage`](GAP_ref(ref:TestPackage)).
+"""
+function test(name::String)
+  global disable_error_handler
+
+  function with_gap_var(f, name::String, val)
+    gname = GapObj(name)
+    old_value = Globals.ValueGlobal(gname)
+    Globals.MakeReadWriteGlobal(gname)
+    Globals.UnbindGlobal(gname)
+    Globals.BindGlobal(gname, val)
+    try
+        f()
+    finally
+      Globals.MakeReadWriteGlobal(gname);
+      Globals.UnbindGlobal(gname);
+      Globals.BindGlobal(gname, old_value);
+    end
+  end
+
+  error_occurred = false
+  ended_using_QuitGap = false
+  function fake_QuitGap(code)
+    if code != 0
+      error_occurred = true
+    end
+    ended_using_QuitGap = true
+  end
+
+  disable_error_handler[] = true
+  result = false
+  try
+    with_gap_var("ERROR_OUTPUT", Globals._JULIAINTERFACE_ORIGINAL_ERROR_OUTPUT) do
+      with_gap_var("QuitGap", fake_QuitGap) do
+        with_gap_var("QUIT_GAP", fake_QuitGap) do
+          with_gap_var("ForceQuitGap", identity) do
+            with_gap_var("FORCE_QUIT_GAP", identity) do
```

Comment:
```suggestion
          with_gap_var("ForceQuitGap", fake_QuitGap) do
            with_gap_var("FORCE_QUIT_GAP", fake_QuitGap) do
```
I think that would make WPE pass

Reply:
Seems reasonable